### ISSUE TITLE
Firestore not showing all list of data in firebase panel

### ISSUE THREAD

**Issue by BhavinBhadani** [firebase/quickstart-ios#430]
Recently I dived into sample firestore and create one sample project for that to run this sample app, it runs successfully and also populates successfully but some why there are only 2 entries display in restaurants document where is populate fields are more than 20 I guess .. why so?

**Comment by morganchen12** [comment]
This is a known issue, a fix should be released soon.

**Comment by BhavinBhadani** [comment]
@morganchen12 I have one question. can we fetch data from other projects firebase documents?

For example if I have 2 app and I want to connect both to each other with firebase, is it possible?

**Comment by morganchen12** [comment]
Yes. You can also have multiple instances of `FirebaseApp` (`FIRApp` in ObjC) running in the same project, so you can have multiple Firestores as well.

**Comment by BhavinBhadani** [comment]
Yes, that's ok to have multiple databases with the same app but can we have access to that database with other project with different bundle id?

**Comment by BhavinBhadani** [comment]
@morganchen12  sir can u please tell me is it possible or not?

**Comment by morganchen12** [comment]
You can access multiple projects from within an app, but not from Firebase Console at the same time.

You can have two separate tabs open, each with one project, but there's no way to combine the project data in console.
